view/frontend/web/js/view/payment/method-renderer/bold-method.js

```diff
@@ -37,38 +38,40 @@
                 this.subscribeToPIGI();
                 this.customerIsGuest = !!Number(window.checkoutConfig.bold.customerIsGuest);
                 this.awaitingRefreshBeforePlacingOrder = false;
-                this.iframeSrc(window.checkoutConfig.bold.payment.iframeSrc);
-                this.syncBillingData();
                 this.messageContainer.errorMessages.subscribe(function (errorMessages) {
                     if (errorMessages.length > 0) {
                         loader.stopLoader();
                     }
                 });
+
+                const sendRefreshOrder = _.debounce(
+                    function () {
+                        this.refreshOrder();
+                    }.bind(this),
+                    500
+                );
+
+                sendRefreshOrder();
+
                 quote.billingAddress.subscribe(function () {
-                    const sendBillingAddress = _.debounce(
-                        function () {
-                            this.syncBillingData();
-                        }.bind(this),
-                        1000
-                    );
-                    sendBillingAddress();
+                    sendRefreshOrder();
                 }, this);
                 const email = registry.get('index = customer-email');
                 if (email) {
                     email.email.subscribe(function () {
                         if (email.validateEmail()) {
-                            const sendGuestCustomerInfo = _.debounce(
-                                function () {
-                                    this.sendGuestCustomerInfo();
-                                }.bind(this),
-                                1000
-                            );
-                            sendGuestCustomerInfo();
+                            sendRefreshOrder();
                         }
                     }.bind(this));
                 }
             },
 
+            initializePaymentGateway: function () {
+                console.log('initializing pigi...');
+                // TODO: Set frame src once /refresh is done
```

Comment:
Is this left over?

Reply:
I'd like to leave the comment, just removing the todo